File: Data Structures & Algorithms/trapping-rain-water/submission-0.java
public class Solution {
    public int trap(int[] height) {
        int output = 0;
        if (height == null || height.length == 0) {
            return 0;
        }
        for (int i = 0; i < height.length; i++) {
            int leftMax = height[i];
            int rightMax = height[i];
        
            for (int j = 0; j < i; j++) {
                leftMax = Math.max(leftMax, height[j]);
            }
            for (int j = i+1; j < height.length; j++) {
                rightMax = Math.max(rightMax, height[j]);
            }
            output = output + Math.min(leftMax, rightMax) - height[i];
        }
        return output;
    }
}

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        output = 0
        for i in range(0, len(prices)):
            for j in range(i+1, len(prices)):
                if prices[j]>prices[i]:
                    output = max(output, prices[j]-prices[i])

        return output